libexec/scoop-info.ps1

```diff
@@ -112,15 +112,15 @@ if ($status.installed) {
 
         # Collect file list from each location
         $appFiles = Get-ChildItem $appsdir -Filter $app
-        $currentFiles = Get-ChildItem $appFiles -Filter (Select-CurrentVersion $app $global)
+        $currentFiles = Get-ChildItem (Join-Path $appsdir $appFiles) -Filter (Select-CurrentVersion $app $global)
```

Comment:
```suggestion
        $currentFiles = Get-ChildItem $appFiles.FullName -Filter (Select-CurrentVersion $app $global)
```

`.FullName` is enough otherwise it fails in PowerShell 7